### PR TITLE
Adding explanation of how to create an svg component in react

### PR DESCRIPTION
**Pull request by cohitre**

### CODE REVIEW

**Comment by cohitre** [comment]
ping

File: README.md [hunk 1]
```diff
@@ -3,3 +3,17 @@
...
+## Adding an SVG icon
+
+1. Remove all attributes except for viewBox and add a className to the `<svg>` tag.
+    const classes = classnames('tilt-icon tilt-icon-SOME_VALUE', this.props.className);
+
+2. Remove `<desc/>` tags and comments.
+3. Set `<title/>` tag text to be the name of the component (this is used in the `"renders without error"` tests).
+4. Add a `tilt-icon-main` class if necessary
+5. Remove all ids and unused attributes. Style attributes will live in the CSS file.
+6. OPTIONAL Consolidate `transforms` into a single group
```
Comment (tildedave) [inline README.md]: could we have an example here?  I know how to do this but only because I played with it once

File: README.md [hunk 2]
```diff
@@ -3,3 +3,18 @@
...
+
+1. Remove all attributes except for viewBox and add a className to the `<svg>` tag.
+    `const classes = classnames('tilt-icon tilt-icon-SOME_VALUE', this.props.className);`
+
+2. Remove `<desc/>` tags and comments.
+3. Set `<title/>` tag text to be the name of the component (this is used in the `"renders without error"` tests).
+4. Add a `tilt-icon-main` class if necessary
+5. Remove all ids and unused attributes. Style attributes will live in the CSS file.
+6. OPTIONAL Consolidate `transforms` into a single group
+    Exporting files from Sketch often creates additional `<g>` elements with a position transform. Often times the transforms can be added together into a single transform in a single `g` element and the empty `g`s can be removed.
```
Comment (tildedave) [inline README.md]: sorry wasn't clear enough here - was thinking something like `e.g, transform(-10, 20)` nested with `transform(20, 40)` becomes `transform(10, 60)`

**Comment by tildedave** [comment]
lgtm with that suggestion

**Pull request merged**